website/theme.config.tsx

```diff
@@ -0,0 +1,232 @@
+          ]}
+        />
+        <List
+          title="& More"
+          items={[
+            { title: "Blog", url: "/blog" },
+            { title: "GraphQL Foundation", url: "/foundation" },
+            {
+              title: "GraphQL Community Grant",
+              url: "/foundation/community-grant",
+            },
+            { title: "Logo and Brand Guidelines", url: "/brand" },
+            { title: "Code of Conduct", url: "/codeofconduct" },
+          ]}
+        />
+      </div>
+      <div className="flex justify-between flex-wrap gap-10">
+        <p className="text-sm">
+          Copyright © 2023 The GraphQL Foundation. All rights reserved.
+          <br />
+          For web site terms of use, trademark policy and general project
+          policies please see{" "}
+          <a
+            href="https://lfprojects.org"
+            target="_blank"
+            rel="noreferrer"
+            className={`text-primary ${classes.link}`}
+          >
+            https://lfprojects.org
+          </a>
+        </p>
+        <ul className="flex gap-5">
+          {[
+            { url: "https://github.com/graphql", icon: GitHubIcon },
+            { url: "https://discord.graphql.org", icon: DiscordIcon },
+            { url: "https://twitter.com/graphql", icon: TwitterIcon },
+            {
+              url: "http://stackoverflow.com/questions/tagged/graphql",
+              icon: StackOverflowIcon,
+            },
+          ].map(({ url, icon: Icon }) => (
+            <li key={url}>
+              <a
+                href={url}
+                target="_blank"
+                rel="noreferrer"
+                className="hover:text-primary transition-colors"
+              >
+                <Icon className="h-5 w-auto *:fill-current" />
+              </a>
+            </li>
+          ))}
+        </ul>
+      </div>
+    </div>
+  )
+}
+
+export default {
+  backgroundColor: {
+    dark: "27,27,27",
+  },
+  head: function useHead() {
+    const { frontMatter, title: pageTitle } = useConfig()
+    const { asPath } = useRouter()
+
+    const title = `${pageTitle}${asPath === "/" ? "" : " | GraphQL"}`
+    const { description, canonical, image } = frontMatter
+    return (
+      <>
+        <title>{title}</title>
+        <meta property="og:title" content={title} />
+        {description && (
+          <>
+            <meta name="description" content={description} />
+            <meta property="og:description" content={description} />
+          </>
+        )}
+        {canonical && <link rel="canonical" href={canonical} />}
+        {image && <meta name="og:image" content={image} />}
+        <meta property="og:image" content="/img/og-image.png" />
+        <meta property="twitter:site" content="@graphql" />
+      </>
+    )
+  },
+  banner: {
+    content: (
+      <>
+        📣 GraphQLConf 2024 • Sept 10-12 • San Francisco •{" "}
+        <NextLink
+          href="/conf"
```

Comment:
This link is broken. Should it be `/conf/2024` ?

Reply:
this link is broken only in prod, because Next.js' rewrites/redirects doesn't work for static exported website https://github.com/dimaMachina/graphql.github.io/blob/1057765e20da6afd9cdc313218dea77c1304c985/website/next.config.js#L43-L50

before releasing new website we'll make all redirects/rewrites declared in next.js config working!